Moderation-Policy.md

```diff
@@ -0,0 +1,48 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
+* "Post" refers to the content of any issue, pull request or comment.
+* "Moderate" refers to the act of modifying the content of or deleting any Post that is in obvious violation of the Node.js Code of Conduct.
+* "Ban" refers to the act of blocking an individual Github account from any further participation in the Node.js Github Organization.
+
+### Policy
+
+* All Posts are expected to respect the Node.js Code of Conduct.
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Collaborator's must not Moderate any Post authored by another Collaborator without first giving the author an opportunity of at least 24 hours to modify or remove the Post on their own. In extreme circumstances the TSC can be consulted to waive the 24 hour grace period.
```

Comment:
- what does "not moderate" mean? does this not mean commenting that a piece of content violates the code of conduct?
- what does extreme mean here?
- are there exceptions for clear trolling? (e.g. the node/inclusivity group's 8chan/4chan situation)
- when does the 24hrs start? from post or fro notification that it needs to be edited/changed? who comments that something needs to be edited/changed?

generally, i am curious what the motivation is here. why not just delete/moderate the comment and the collaborator can repost something more constructive/appropriate?